apps/jump-page/src/components/app/app.component.tsx

```diff
@@ -0,0 +1,78 @@
+import { SlashID } from "@slashid/slashid";
+import { init, ErrorBoundary } from "@sentry/react";
+import { Logo, TextProvider, ThemeRoot } from "@slashid/react-primitives";
+
+import {
+  AppContext,
+  initialAppContextState,
+  type AppState,
+  type AppContextState,
+  type AppReadyState,
+} from "./app.context";
+import { useEffect, useMemo, useRef, useState } from "react";
+import { I18N, detectLanguage } from "../../domain/i18n";
+import { Flow, Error } from "../flow";
+import { Card } from "../card";
+
+import * as styles from "./app.css";
+import { config } from "../../config";
+
+export function App() {
+  const isBrowser = !!globalThis.window;
```

Comment:
@ikovic we have something like this in other package, IDK if it makes much difference in that case
```jsx
export function isBrowser(): boolean {
  return typeof window !== "undefined" && globalThis === window;
}
```

Reply:
Yep, this one is available from the react primitives package. I'll see if I can use it instead of reimplementing it here. Thanks!

Reply:
Resolved!